src/apps/CONVECTION3DPA.hpp

```diff
@@ -378,14 +378,18 @@ class CONVECTION3DPA : public KernelBase
   void runCudaVariant(VariantID vid, size_t tune_idx);
   void runHipVariant(VariantID vid, size_t tune_idx);
   void runOpenMPTargetVariant(VariantID vid, size_t tune_idx);
+  void runSyclVariant(VariantID vid, size_t tune_idx);
 
   void setCudaTuningDefinitions(VariantID vid);
   void setHipTuningDefinitions(VariantID vid);
+  void setSyclTuningDefinitions(VariantID vid);
 
   template < size_t block_size >
   void runCudaVariantImpl(VariantID vid);
   template < size_t block_size >
   void runHipVariantImpl(VariantID vid);
+  template < size_t block_size >
```

Comment:
I've been naming the template parameter for the SYCL methods `work_group_size`, in the header and implementation files, since that is the SYCL terminology. Please do that for consistency.